integration/test/Test/FeatureFlags/ConferenceCalling.hs

```diff
@@ -24,3 +27,52 @@ testConferenceCalling access = do
     & addUpdate (confCalling def {sft = toJSON True})
     & addUpdate (confCalling def {sft = toJSON False})
     & addInvalidUpdate (confCalling def {sft = toJSON (0 :: Int)})
+
+checkPatchConferenceCalling ::
```

Comment:
Maybe this could be abstracted a bit to avoid copy-pasting the whole thing? One idea could be to pass a function that computes the expected status (or maybe the whole expected value of the feature).